.github/workflows/swift.yml

```diff
@@ -5,10 +5,11 @@ name: Swift
 
 on:
   push:
-    branches: [ "main", "mock-server-integration" ]
+    branches: [ "main"]
```

Comment:
Pushing to `main` should be disabled.  Via PRs only.  Is it not disabled for this repo?  (See your repo settings.)

Reply:
I already have set up branch rules to prevent directly pushing to the main branch.